### ISSUE TITLE
Bug: Example continues to next line

### ISSUE THREAD

**Issue by Ainali** [egils-consulting/LexUse#18]
In [this edit](https://www.wikidata.org/w/index.php?title=Lexeme:L32560&diff=1352261016&oldid=1066299622) the example loses the question mark and picks up the number from the next numbered point in the list in the [source document](https://data.riksdagen.se/dokument/H4B52). I would have expected that the last question mark would stay and marked the end of the phrase.

**Comment by dpriskorn** [comment]
Yeah, thats because the scrips detection of what is a sentence splits along the full stops ".". I normally say no to a sentence like that that is incomplete.
Splitting the text from Riksdagen properly into sentences would probably require training an AI. I have not learned now to do that yet, but python has good support for that should we want.
I would very much prefer if upstream could do that work and deliver better search results with say
Sentence with match, before_context and after_context. Right now we get a cut out section of a document and no indication of sentence borders.

**Comment by dpriskorn** [comment]
Also I don't like questions as example sentences, so a NLP AI as exist for English would be very nice to reliably exclude those.

**Comment by dpriskorn** [comment]
I don't know why it doesn't interpret "?" as a sentence border. It's using a regex I found that I did not care to improve. If you feel like it feel free to experiment with a regex that works better, but we need an NLP AI that sees more than just text strings to really get somewhere. 
No Swedish authority or organization publish training data nor AI as a downloable binary module for e.g. python. So essentially we are way behind English.

**Comment by dpriskorn** [comment]
See e.g. https://stanfordnlp.github.io/CoreNLP/ where sv is missing

**Comment by dpriskorn** [comment]
This might help and with the new dataset from AF we have more text.
https://github.com/af-ai-center/SweBERT

**Comment by Ainali** [comment]
I didn't find an example now, but I have also encountered examples like "Herr talman! Jag vill gratulera min debattmotståndare." where the form was something in the second sentence. There I would expect that "Herr talman!" would not be part of the text for the example.

**Comment by dpriskorn** [comment]
Thanks, so the regex is eh broken.
